src/main/resources/db/mysql/data.sql

```diff
@@ -4,16 +4,21 @@ INSERT IGNORE INTO vets VALUES (3, 'Linda', 'Douglas');
 INSERT IGNORE INTO vets VALUES (4, 'Rafael', 'Ortega');
 INSERT IGNORE INTO vets VALUES (5, 'Henry', 'Stevens');
 INSERT IGNORE INTO vets VALUES (6, 'Sharon', 'Jenkins');
+INSERT IGNORE INTO vets VALUES (7, 'Eugene', 'Shpak');
```

Comment:
Not necessary as we do not use MySQL.

Reply:
if so, then let's delete all unnecessary resources

Reply:
You will do it in a separate pull request.